Mathlib/Data/Real/StarOrdered.lean

```diff
@@ -0,0 +1,31 @@
+/-
+Copyright (c) 2020 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Floris van Doorn, Yury Kudryashov
```

Comment:
@semorrison, I think you can claim this file thanks to leanprover-community/mathlib#4686

Reply:
Sure. I've been (and plan to continue!) to just copy and paste the header from whichever file I am splitting out of, but have no objections if others want to do the forensic work. :-)

Reply:
Oh, I guess in this one I didn't and just made a new one for some very small files.